mail_drop_target/tests/test_mail_drop_target.py

```diff
@@ -0,0 +1,75 @@
+import base64
+
+from mock import patch
+
+from odoo import exceptions, tools
+from odoo.tests.common import TransactionCase
+
+
+class TestMailDropTarget(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.partner = self.env["res.partner"].create({"name": "TEST PARTNER"})
+        self.partner.message_subscribe(partner_ids=self.partner.ids)
```

Comment:
```suggestion
class TestMailDropTarget(SavepointCase):
    @classmethod
    def setUpClass(cls):
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
        cls.partner = cls.env["res.partner"].create({"name": "TEST PARTNER"})
        cls.partner.message_subscribe(partner_ids=cls.partner.ids)
```
otherwise:
1. partner is re-created for every test
2. tracking will slow down every create/write

Reply:
Thanks!